man/rgblink.1

```diff
@@ -78,7 +78,8 @@ If specified, the map file will not list symbols, only sections.
 .It Fl m Ar map_file , Fl \-map Ar map_file
 Write a map file to the given filename, listing how sections and symbols were assigned.
 .It Fl n Ar sym_file , Fl \-sym Ar sym_file
-Write a symbol file to the given filename, listing the address of all exported symbols.
+Write a symbol file to the given filename, listing all exported symbols.
```

Comment:
Actually, this lists more than just *exported* symbols... What's the rule exactly?

Reply:
I'm pretty sure it is in fact limited to exported symbols, of type `SYMTYPE_EXPORT`.

Reply:
There are a lot of counter-examples. A lot of local labels are not exported (nor implicitly via `-E`), yet appear in the sym file.

Reply:
That's true about labels; I'll reword the docs.

This now outputs non-label constants from the `symbols` vector. That is only ever added to via the `sym_AddSymbol` function, which is called in object.cpp and sdas_obj.cpp. object.cpp checks if `symbol.type == SYMTYPE_EXPORT` before adding it, and sdas_obj.cpp sets `symbol.type = SYMTYPE_EXPORT` before adding it, so these are guaranteed to be exported.